src/test/java/uk/gov/companieshouse/confirmationstatementapi/checks/CompanyPscCheckTest.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.companieshouse.confirmationstatementapi.checks;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import uk.gov.companieshouse.api.model.psc.PscsApi;
+import uk.gov.companieshouse.confirmationstatementapi.exception.ServiceException;
+import uk.gov.companieshouse.confirmationstatementapi.service.PscService;
+
+@ExtendWith(MockitoExtension.class)
+public class CompanyPscCheckTest {
+
+    private static final String COMPANY_NUMBER = "12345678";
+    
+    @Mock
+    private PscService pscService;
+
+    private PscsApi pscsApi;
+    private CompanyPscCheck check;
+    
+    @BeforeEach
+    void setUp() {
+        check = new CompanyPscCheck(pscService);
+        pscsApi = new PscsApi();
+    }
+    
+    @Test
+    void checkForMultiplePscsTest() throws ServiceException {
+        pscsApi.setActiveCount((long) 3);
```

Comment:
instead of `(long) 3`, you can just put `3L`, ever so slightly more efficient as `(long) 3` creates an int, and then transforms it to a long. But `3L` just creates a long in the first place